### ISSUE TITLE
Whitespace character between nodes prevents loading.

### ISSUE THREAD

**Issue by Walamazoo** [YarnSpinnerTool/YarnSpinner#331]
**What is the current behavior?**

Having a whitespace character between nodes, such as the ones created by the space and tab keys, causes following nodes to throw Yarn.DialogueException: No node named _____ has been loaded. No error appears in the inspector, only at runtime.

**Please provide the steps to reproduce, and if possible a minimal demo of the problem**:

```
title: Start
---
This is the start node.
<<jump Example>>
===
 
title: Example
---
This is an example node.
===
```
Will throw Yarn.DialogueException: No node named Example has been loaded at runtime because there is a whitespace character made by the space key between lines 5 and 7. 

**What is the expected behavior?**

Having a space between nodes should cause a token recognition error or extraneous input ' ' expecting HEADER_DELIMITER error.

**Please tell us about your environment:**
  
  - Yarn Spinner Version: v2.2.15
  - Unity Version: 2020.3.25f1

**Other information** 

https://en.wikipedia.org/wiki/Whitespace_character#:~:text=When%20rendered%2C%20a%20whitespace%20character,word%20divider%20in%20Western%20scripts.

**Comment by desplesda** [comment]
This is a strange one. I've been looking at this bug, and it appears that the compiler just _stops_ at the space, rather than generating errors. Will keep looking at it.

**Comment by McJones** [comment]
closed in 8af11cc
